src/settings/backendsettingspage.ui

```diff
@@ -162,6 +162,19 @@
       <string>Options</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
+      <item>
+       <widget class="QRadioButton" name="radioButton_disable_audio_normalization">
```

Comment:
Lowercase widget name

Reply:
This along with "Replay gain" and "EBU R 128 Loudness Normalization" groupboxes should be in a groupbox called "Audio normalization". Right now this radiobutton looks  out of place.

Reply:
I tested this out, looks much better, see: https://github.com/strawberrymusicplayer/strawberry/commit/0ff36e8322d1b965a5c36b20ddd3632a1262c91b
You can cherry-pick that commit.

Reply:
Thanks, much nicer!